### DISCUSSION TITLE
download schema to include currently accepted name or species ID

### DISCUSSION THREAD

**Discussion by Calflora** [Symbiota/community-board#358] [Wish List]
I'm downloading California bryophyte data periodically from the bryoportal through the symbiota schema.
The most difficult part of this process is dealing with old names.
Would it be possible for you to provide an alternate schema that includes the currently accepted name or species ID for each record?

Thank you

**Comment by themerekat** [comment]
In the Bryophyte Portal, this might be a feasible option since the Bryophyte taxonomic thesaurus is relatively well-maintained; however, most Symbiota portals can't be considered taxonomic authorities. Therefore, it's probably more realistic to align the taxonomy after downloading based on a desired taxonomic resources (e.g., using the Taxonomic Name Resolution Service https://tnrs.biendata.org/).
